src/components/common/types.ts

```diff
@@ -1 +1,6 @@
 export type Direction = 'ltr' | 'rtl' | 'auto';
+
+export type IgniteComponent = CustomElementConstructor & {
```

Comment:
Is this making it to the root export? Can't quite see it

Reply:
Sure, I will export it